client/network/transactions/src/lib.rs

```diff
@@ -175,23 +157,25 @@ impl TransactionsHandlerPrototype {
 	/// the behaviour of the handler while it's running.
 	///
 	/// Important: the transactions handler is initially disabled and doesn't gossip transactions.
-	/// You must call [`TransactionsHandlerController::set_gossip_enabled`] to enable it.
-	pub fn build<B: BlockT + 'static, H: ExHashT>(
+	/// You must call `TransactionsHandlerController::set_gossip_enabled` to enable it.
```

Comment:
It looks like this was replaced with a call to `service.is_major_syncing()` and the comment needs changing.

Reply:
Fixed